styles.css

```diff
@@ -610,3 +626,486 @@ li {
 #button_contact:active {
   background-color: #008552;
 }
+
+@media all and (min-width: 768px) {
+  .header2 {
+    display: none;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  body {
+    width: 1440px;
```

Comment:
I think because of the static width size, it is not aligning all of your content in the center. Please kindly make sure that the content of your entire application is positioned in the center.  Following sections should be positioned in the middle of the screen:
1. Introduction
2. Recent Works
3. About Me
4. Contact  
Kindly consider fixing this, please. 👍🏽 

<img width="1440" alt="Screenshot 2023-01-21 at 10 51 13 AM" src="https://user-images.githubusercontent.com/24832846/213845004-a63bfa27-7156-456a-854f-aade2664801d.png">